discord_bot/MessageHandler.py

```diff
@@ -0,0 +1,35 @@
+import WikipediaHttpClient as wiki_client
+
+SEARCH = "/search"
+HELP = "/help"
+
+HELP_MESSAGE = " # SAMPLE HELP MESSAGE #"
+ERROR_INFO = "Cannot call Wikipedia API: "
+
+
+async def handle_message(message, client):
+    if message.author == client.user:
+        return
```

Comment:
Does it mean, that in case if I'll add to `.env` my token, the bot will not respond to a message?